charts/rbs/templates/deployment.yaml

```diff
@@ -0,0 +1,138 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: terra-rbs-deployment
+  labels:
+    {{ template "rbs.labels" . }}
+spec:
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: terra-rbs
+  template:
+    metadata:
+      name: terra-rbs-service
+      labels:
+        app: terra-rbs
```

Comment:
Do we want to include the other ```{{ template "rbs.labels" .  }}``` here in addition so that pods will have those labels too?

Reply:
Done